iulianIachimov/src/main/java/genericstask/model/SimpleLibrary.java

```diff
@@ -59,23 +59,27 @@ public void printVideos(List<Video> videoList) {
   }
 
     public void printNewsPaper(List<Newspapers> newspaperList) {
-        for (Newspapers newsPaper : newspaperList) {
-            System.out.println(newsPaper.getTitle() + " in " + newsPaper.getLanguage());
-        }
-        System.out.println();
+      for (Newspapers newsPaper : newspaperList) {
+        System.out.println(newsPaper.getTitle() + " in " + newsPaper.getLanguage());
+      }
+      System.out.println();
```

Comment:
( • - •)

Reply:
Sorry, I do not understand what needs to be done here

Reply:
Fixed